api/v1beta2/ibmvpccluster_types.go

```diff
@@ -94,6 +98,24 @@ type AdditionalListenerSpec struct {
 	Port int64 `json:"port"`
 }
 
+// VPCNetworkSpec defines the desired state of the network resources for the cluster.
+type VPCNetworkSpec struct {
+	// computeSubnetsSpec is a set of Subnet's which define the Compute subnets.
+	// +optional
+	ComputeSubnetsSpec []Subnet `json:"computeSubnetsSpec,omitempty"`
+
+	// controlPlaneSubnetsSpec is a set of Subnet's which define the Control Plane subnets.
+	// +optional
+	ControlPlaneSubnetsSpec []Subnet `json:"controlPlaneSubentsSpec,omitempty"`
```

Comment:
any specific reason for having different subnet for the controlplane and worker nodes?

Reply:
Yes, the alternative is adding a field to the Subnet to be worker versus control plane. I think it is easier to handle with separate lists.